### PR TITLE
Document library

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Add the document library feature

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 2cc3d8b |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-336.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by lbadenhop** [comment]
@cbordier, is the Documents management topic ready for review?

**Comment by cbordier** [comment]
@lbadenhop yes you can start review my poor english ;-).
But wait for the merge as it is not pushed to production yet.

File: content/en/docs/manage_document_library/documentation_library_management.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+3. Select the method **Upload Document(s)**, and either use the **BROWSE FILES** button or drop file(s) in the drop area. The successfully uploaded file is displayed in the Manage Uploads area. If a file does not respect the limitations, it will appear in an error message area.
+4. Check the box in front of the uploaded file to edit the file details: Document name, Description, Version. Once your document information is correct, click **Next**.
+5. Add your document visibility: Product and/or Marketplace. This visibility will be applied to all documents. If you need specific visibility for a document, consider uploading only that document. Then click **Next**.
+6. Add the team permissions: Restricted (only the Central Admin user can manage the document) / Shared (specific team permission: owner / read / edit / none). These permissions will be applied to all documents. If you need specific permissions for a document, consider uploading only that document. Then click **Next**
+7. Review your uploaded files. You can return to any page using either the ellipsis menu next to the document tile or the **Prev** button. Once everything looks good click **Save**.
```
Comment (lbadenhop) [inline content/en/docs/manage_document_library/documentation_library_management.md:85]:
```suggestion
7. Review your uploaded files. You can return to any page using either the ellipsis menu next to the document tile or the **Prev** button. Once everything looks good click **Save**.
```

Comment (lbadenhop) [inline content/en/docs/manage_document_library/documentation_library_management.md:85]: The ellipsis menu deletes the file. The Prev button takes you back to previous pages. Maybe the ellipsis menu isn't working correctly in https://platform.axwaytest.net/org/marketplace?

File: content/en/docs/manage_document_library/documentation_library_management.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+## Create a document from existing files
+
+Only PDF, MD, DOC, DOCX, XLSX and PPTX files are accepted. All files larger than 5MB will be rejected.
+
+To create a document:
+
+1. Navigate to *Document Library*.
+2. Click **+ Add New Document** to open the documents wizard.
+3. Select the method **Upload Document(s)**, and either use the **BROWSE FILES** button or drop file(s) in the drop area. The successfully uploaded file is displayed in the Manage Uploads area. If a file does not respect the limitations, it will appear in an error message area.
```
Comment (lbadenhop) [inline content/en/docs/manage_document_library/documentation_library_management.md:81]: Did not test an unsuccessful upload, so didn't verify whether it shows up in an error message area.

Comment (cbordier) [reply content/en/docs/manage_document_library/documentation_library_management.md:81]: This is what it shows:
image

File: content/en/docs/manage_document_library/documentation_library_management.md (hunk 1 above)
Comment (cbordier) [reply content/en/docs/manage_document_library/documentation_library_management.md:85]: Yes ellipsis menu is limited to delete at the moment.

Comment (lbadenhop) [reply content/en/docs/manage_document_library/documentation_library_management.md:85]: Then I should change step 7 to remove the use of the ellipsis menu to go back to previous pages?

**Comment by lbadenhop** [comment]
@cbordier, please review my edits. Also, can you resolve the conflicts?

**Review by lbadenhop — APPROVED** [review]

**Comment by cbordier** [comment]
@lbadenhop conflict is resolved.
You can merge as the UI deployment is in progress.

**Pull request merged**